galleries/examples/showcase/pan_zoom_overlap.py

```diff
@@ -0,0 +1,74 @@
+"""
+===================================
+Pan/zoom events of overlapping axes
+===================================
+
+Example to illustrate how pan/zoom events of overlapping axes are treated.
+
+
+The default is the following:
+
+- Axes with a visible patch capture pan/zoom events
+- Axes with an invisible patch forward pan/zoom events to axes below
+- Shared axes always trigger with their parent axes
+  (irrespective of the patch visibility)
+
+Note: The visibility of the patch hereby refers to the value of
+``ax.patch.get_visible()``. The color and transparency of a
+patch have no effect on the treatment of pan/zoom events!
+
+
+``ax.set_forward_navigation_events(val)`` can be used to override the
+default behaviour:
+
+- ``True``:  Forward navigation events to axes below.
+- ``False``: Execute navigation events only on this axes.
+- ``"auto"``: Use the default behaviour.
+
+To disable pan/zoom events completely, use ``ax.set_navigate(False)``
+
+"""
+
+
+import matplotlib.pyplot as plt
+
+f = plt.figure(figsize=(11, 6))
+f.suptitle("Showcase for pan/zoom events on overlapping axes.")
```

Comment:
We usually use `fig`:
```suggestion
fig = plt.figure(figsize=(11, 6))
fig.suptitle("Showcase for pan/zoom events on overlapping axes.")
```